site/docs/components/progress/examples.mdx

```diff
@@ -26,45 +26,36 @@ Use linear or circular depending on the context, layout and functionality of an
 
 </LivePreview>
 
-<LivePreview componentName="progress" exampleName="HiddenLabel" displayName="Hide label">
+<LivePreview componentName="progress" exampleName="LinearIndeterminate" displayName="Indeterminate linear progress">
 
-## Hide label
+## Indeterminate linear progress
 
-Use the `hideLabel` prop to display the progress without a label.
+Use an indeterminate linear progress to display an unspecified wait time such as a loading state.
```

Comment:
@navkaur76 will this cause confusion between loading e.g. spinner and loading of a determinate progress state?

Reply:
Is there any more guidance we can give.

Reply:
Based on the duration foundation, an indeterminate linear progress would fall between notable and cut-off, the same as spinner. However, the difference is a linear progress would eventually provide indication of how long a system may take to respond once it goes from indeterminate > determinate. Could we possibly amend it to something like "Use an indeterminate linear progress to indicate the system is processing or loading an action, with the intention of providing a determined wait time before the cut-off point." What do you think? @joshwooding

Reply:
Added additional line before Nav's comment, can combine if needed

Reply:
@navkaur76 updated copy